### DISCUSSION TITLE
Support models other than anthropic using AWS Bedrock

### DISCUSSION THREAD

**Discussion by alanktso** [cline/cline#543] [Feature Requests]
We can now only choose models from Anthropic when using AWS Bedrock. It would be great if we can pick other models such as Meta's Llama.

**Comment by shrijayan** [comment]
https://github.com/cline/cline/discussions/1208

In AWS Bedrock, they support prompt caching through Prompt Routers, which have implemented caching for Anthropic and Meta models. They provide the model ARN ID to access the models. By implementing **Accept Model ARN ID in AWS Bedrock Model Selection #1208**, where we can input the model ARN ID in AWS Bedrock, this feature will be highly useful.

**Comment by SebastianAwaze** [comment]
Yes, I would also like to be able manually input a model. This will be needed especially with fine tuned agents.
